### ISSUE TITLE
vagrant up failing in /deployment/vagrant-development/

### ISSUE THREAD

**Issue by kevmoo** [TechEmpower/FrameworkBenchmarks#1770]
at 48f01686978f853d168ffd2ff2a055de643648fe

```
==> default: cat:
==> default: /home/vagrant/.ssh/authorized_keys
==> default: : Permission denied
==> default: Installing all software
==> default: Traceback (most recent call last):
==> default:   File "toolset/run-tests.py", line 11, in <module>
==> default:
==> default: from benchmark.benchmarker import Benchmarker
==> default:   File "/FrameworkBenchmarks/toolset/benchmark/benchmarker.py", line 34, in <module>
==> default:
==> default: import progressbar
==> default: ImportError
==> default: :
==> default: No module named progressbar
==> default: Setting up welcome message
```

**Comment by asilvas** [comment]
Any luck? Having the same issue.

**Comment by kevmoo** [comment]
Nope :confused:

**Comment by johnrengelman** [comment]
Also having the same issue.

**Comment by johnrengelman** [comment]
I worked around this by doing the following:
```bash
$ vagrant up //Let this complete and fail with the error above
$ vagrant ssh
$ cd ~/FrameworkBenchmarks/toolset/setup/linux
$ ./prerequisites.sh
$ exit
$ vagrant provision
```

**Comment by hamiltont** [comment]
This should be easy enough to fix, just move all the dependencies back into the (de facto standard) `requirements.txt` file instead of specifying them one by one in the `.travis.yml`

**Comment by msmith-techempower** [comment]
@ssmith-techempower Let's squeeze this into #1786 while you're working on it.

**Comment by johnrengelman** [comment]
I'll give it a try again today on latest master.

**Comment by ikkez** [comment]
same problem here.

**Comment by msmith-techempower** [comment]
I BELIEVE this was resolved in #1802 

See the comment in that pull request regarding file mirroring, but vagrant SHOULD be working again.
